applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -161,6 +248,18 @@ public static ConfigurationAsyncClientBuilder builder() {
      *
      * If {@link ConfigurationSetting#etag() etag} is specified, the configuration value is only updated if it matches.
      *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Update the setting with the key-label pair "prodDBConnection"-"westUS" to have the value "updated_db_connection".</p>
+     *
+     * <pre>
+     * client
+     *     .updateSetting(new ConfigurationSetting().key("prodDBConnection").label("westUS").value("db_connection"))
```

Comment:
Your text on line 253 says the value will be updated to "updated_db_connection", but that isn't what you're passing in to the method call.